src/api/data_source_routes.ts

```diff
@@ -25,14 +29,18 @@ export default class DataSourceRoutes {
         app.delete("/containers/:id/import/datasources/:sourceID/active",...middleware, authInContainer("read", "data"),this.setInactive);
 
         app.get("/containers/:id/import/datasources/:sourceID/imports",...middleware, authInContainer("read", "data"),this.listDataSourcesImports);
-        app.post("/containers/:id/import/datasources/:sourceID/imports",...middleware, authInContainer("write", "data"),this.createManualJsonImport);
+        app.post("/containers/:id/import/datasources/:sourceID/imports",...middleware, fileUpload({limits:{fileSize: 50 * 1024 *1024}}), authInContainer("write", "data"),this.createManualJsonImport);
```

Comment:
until we rewrite the json import using busboy we must use this middleware. I can't include this on the main router anymore as it interferes with `busboy`. Moved it from the main router to the route that needs it.